docs/md/modules-wgs.md

```diff
@@ -0,0 +1,319 @@
+# ExternalWholeGenomeReprocessing workload
+
+## Inputs
+An `ExternalWholeGenomeReprocessing` workload
+specifies the following inputs
+for each workflow:
+
+- input_cram or input_bam (required)
+- sample_name (required)
```

Comment:
Can you remove the required bit from the `sample_name` please? Sorry, I forgot to update this!

Reply:
Sure!